app/src/lib/stores/app-store.ts

```diff
@@ -5523,6 +5524,47 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     await this.currentBranchPruner.testPrune()
   }
+
+  public async _showCreateforkDialog(
+    repository: RepositoryWithGitHubRepository
+  ) {
+    const account = getAccountForRepository(this.accounts, repository)
+    if (account === null) {
+      return
+    }
+    await this._showPopup({
+      type: PopupType.CreateFork,
+      repository,
+      account,
+    })
+  }
+
+  /**
+   * Converts a local repository to use the given fork
+   * as its default remote and associated `GitHubRepository`.
+   */
+  public async _convertRepositoryToFork(
```

Comment:
Shouldn't this also add the current default remote as we do when cloning a regular fork? See `addUpstreamRemoteIfNeeded` and `updateExistingUpstreamRemote`. I suspect it'll eventually get added automatically but we should set it up correctly from the start.

Reply:
ah yes, thank you! 5446528da